app/models/cloud_service.rb

```diff
@@ -0,0 +1,37 @@
+class CloudService < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :host
+  belongs_to :system_service
+  belongs_to :availability_zone
+
+  def fog_service
+    connection_options = {:service => source}
+    ext_management_system.with_provider_connection(connection_options) do |service|
+      service.services.find { |s| s.id.to_s == ems_ref }
```

Comment:
OpenStack itself currently do not support that API call (http://developer.openstack.org/api-ref-compute-v2.1.html#os-services-v2.1)

There is a blueprint (https://blueprints.launchpad.net/nova/+spec/get-service-by-id) but spec proposal patch is abandoned (https://review.openstack.org/#/c/172412/).

Maybe we can try to move the ```find``` call into https://github.com/fog/fog-openstack but that would be the same ruby call just at another place.

Creating 'fake' GET/service call in [Fog OpenStack API](https://github.com/fog/fog-openstack/tree/8b1345380f1186cda4fbed83cc9bdc37042b8c6d/lib/fog/openstack/requests/compute) does not sound like a good idea on the first thought but possible for sure - what do you think @Ladas ?